boa/src/exec/operator/mod.rs

```diff
@@ -118,7 +118,8 @@ impl Executable for BinOp {
                     let v_a = interpreter
                         .realm()
                         .environment
-                        .get_binding_value(name.as_ref());
+                        .get_binding_value(name.as_ref())
+                        .expect("Identifier was not initialized");
```

Comment:
Should we not throw a `ReferenceError` here?

Reply:
You are correct. I made a fix, check latest commit.